src/Dfe.PlanTech.Domain/Content/Models/ContentSupport/Mapped/Types/CustomComponentType.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Dfe.PlanTech.Domain.Content.Models.ContentSupport.Mapped.Types;
+
+public enum CustomComponentType
+{
+    Undefined,
+    Accordion,
+    Attachment,
+    Card,
+    GridContainer,
+    CSAccordion,
+    csCard,
```

Comment:
I'm guessing the inconsistency is just because this is what it is in Contentful? but this seems like a good way to deal with it

Reply:
Yeah exactly, the different cases really annoy me though!

Reply:
Likewise 😄 I think this is the way to go though! nice